src/scss/components/_data-tables.scss

```diff
@@ -228,10 +228,54 @@ $md-data-table-card-header-font-size: 16px !default;
 
     &--responsive {
       overflow-x: auto;
```

Comment:
I think I would prefer to have this extracted into a mixin as well eventually: `react-md-data-tables-fixed` or something.